modules/spark-wallet.nix

```diff
@@ -63,11 +64,9 @@ in {
         User = "clightning";
         Restart = "on-failure";
         RestartSec = "10s";
-        PrivateTmp = "true";
-        ProtectSystem = "full";
-        NoNewPrivileges = "true";
-        PrivateDevices = "true";
-      };
+      } // nix-bitcoin-services.defaultHardening
+        // nix-bitcoin-services.node
+        // nix-bitcoin-services.allowTor;
```

Comment:
This prevents spark-wallet from reaching bitstamp.net to query Bitcoin price in USD. I don't think this is an issue because Unit of Account is Bitcoin not USD.